Orange/distance/__init__.py

```diff
@@ -72,13 +76,15 @@ def __call__(self, e1, e2=None, axis=1, impute=False):
             x1 = x1.T
             if x2 is not None:
                 x2 = x2.T
-        dist = skl_metrics.pairwise.pairwise_distances(x1, x2, metric=self.metric)
+        dist = skl_metrics.pairwise.pairwise_distances(x1, x2,
+                                                       metric=self.metric)
         if isinstance(e1, data.Table) or isinstance(e1, data.RowInstance):
             dist = DistMatrix(dist, e1, e2, axis)
         else:
             dist = DistMatrix(dist)
         return dist
 
+
```

Comment:
Although PEP8 in principle recommends two lines on the module level, do not add empty line here: these lines belong to the class above.